plugins/module_utils/module_helper.py

```diff
@@ -93,6 +93,8 @@ def printf_fmt(_fmt, v):
             self.arg_format = (self.stars_deco(stars))(self.arg_format)
 
     def to_text(self, value):
+        if value is None:
+            return []
```

Comment:
Is this a change that should be backported to stable-1? If yes, please move it to a separate PR.

Reply:
It is, but removing that breaks this PR. Moving to WIP.

Reply:
Solved after merging #2024